### DISCUSSION TITLE
<title> Dns propergation

### DISCUSSION THREAD

**Discussion by Derrick99234** [community/community#55835] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I hosted my website on GitHub and bought a domain name from hostking and the only thing I have access to is nameserver not dns records.
So I remain the default nameserver to
ns1.github.io
ns2.github.io
But it not yet working since yesterday. Please help

**Comment by RevertIT** [comment]
How come you can't edit domain records?

Then ask for a transfer code and transfer your domain to some other reputable domain service.

**Comment by Derrick99234** [comment]
Thank You for your response

I looked for dns checker online, it appear that it's tick in some country.
Now my dear is that
ns1.github.io is it the right nameserver to Link?
Cause it was chatgbt a chat bot that gave me this

**Comment by RevertIT** [comment]
GitHub does not host nameservers for use by users. Set them to the default ones provided by your domain hosting, then set setup A and AAA records.
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

**Comment by Derrick99234** [comment]
Thanks, I'll check it out.

i link the nameserver to cloudfare and link the dns record please check if
it correct
